logical_router_static_route.go

```diff
@@ -172,6 +180,7 @@ func (odbi *ovndb) rowToLogicalRouterStaticRoute(uuid string) *LogicalRouterStat
 		UUID:       uuid,
 		IPPrefix:   cacheLogicalRouterStaticRoute.Fields["ip_prefix"].(string),
 		Nexthop:    cacheLogicalRouterStaticRoute.Fields["nexthop"].(string),
+		Options:    cacheLogicalRouterStaticRoute.Fields["options"].(libovsdb.OvsMap).GoMap,
```

Comment:
With older OVN NB schema this code would panic. We need to add a check here to make sure the field exists (not nil).